tutorials/scripting/scene_unique_nodes.rst

```diff
@@ -40,3 +40,71 @@ name in the path for ``get_node()``. For example:
  .. code-tab:: csharp
 
     GetNode<Button>("%RedButton").Text = "Hello";
+
+Same-scene limitation
+---------------------
+
+A scene unique node can only be retrieved by a node inside the same scene.
+
+If the same scene is instantiated twice, a node inside the first instance
+calling ``get_node("%RedButton")`` will get the button in the first instance,
+not the button in the second instance.
+
+This limitation is perfect for a UI scene like a health bar, where each instance
+of the health bar must not interact with other instances.
```

Comment:
I'm not sure this is worth a mention. This is just how node paths work. You can right the same for any node path within two instances of the same scene. Maybe there is a point to make here, but I'm only confused by the focus of these two paragraphs.

Reply:
The point is that if the scene unique name were global, then the two instances would interact. The implications of having scene unique nodes cached in each scene aren't obvious to everybody, so it seems worth spelling this out a little. And after understanding all this, someone still might want to file a proposal to make scene unique nodes global, so I'm trying to predict that a bit and show why this behavior is actually desirable.

Reply:
Ended up removing. It actually might be nifty to support getting a scene unique node from a nested scene, who knows?

Also removed the next paragraph. Doesn't seem necessary to word it all out with the example below.